build.gradle

```diff
@@ -83,7 +83,7 @@ subprojects {
         jacksonVersion = "2.9.10"
         jacksonDatabindVersion = "2.9.10"
         awsJavaSdkVersion = "1.11.686"
-        guavaVersion = "19.0"
+        guavaVersion = "30.0-android"
```

Comment:
This version is for Android project? `-jre` should be good.

Reply:
The reason of the version is simply following the implicit version upgrade as following.
```
|    |    |    +--- com.fasterxml.jackson.datatype:jackson-datatype-guava:2.9.10
|    |    |    |    +--- com.google.guava:guava:18.0 -> 30.0-android
```
But yes. there is no reason to choose `-android`. So I will check and try to set `-jre`.

Reply:
```
 |    |    |    |    +--- com.google.guava:guava:18.0 -> 30.0-android
```

How did you get the result? I checked v0_11 dependencies and got the following one. But I didn't get 30-android.
```
|    \--- com.google.inject.extensions:guice-multibindings:4.2.2
|         \--- com.google.inject:guice:4.2.2
|              +--- javax.inject:javax.inject:1
|              +--- aopalliance:aopalliance:1.0
|              \--- com.google.guava:guava:25.1-android (*)
```

BTW `com.google.inject:guice:4.2.2` explicitly specifies `25.1-android`
https://github.com/google/guice/blob/86a88f0728ca477b4e33e3c1cb042e0ec02f2feb/pom.xml#L144-L148 and I found a related issue https://github.com/google/guice/issues/1207.

Latest Guice specifies `30.1-jre`
https://github.com/google/guice/blob/master/pom.xml#L144-L148, so `-jre` seems okay.

My concern is `-jre` and `-android` have some differences and it may cause a problem in the future.

Reply:
@komamitsu I guess recent change in `master` branch may affect the guava version.
And agree with `-jre`.
 I will update to the latest `30.1.1-jre`.

https://mvnrepository.com/artifact/com.google.inject/guice/4.2.2

Reply:
Updated the guava version.

Reply:
I checked the dependency of latest `master` branch again, but didn't get `30.0-android`. I want to just avoid hidden unexpected dependency. My operation to check the dependency was like this
```
$ ./gradlew -p digdag-core dependencies | less
(search for "com.google.guava:guava")
  :
|    \--- com.google.inject.extensions:guice-multibindings:4.2.2
|         \--- com.google.inject:guice:4.2.2
|              +--- javax.inject:javax.inject:1
|              +--- aopalliance:aopalliance:1.0
|              \--- com.google.guava:guava:25.1-android (*)
```

Is it different from yours?

Reply:
I guess `digdag-standards` will update guava version.

Reply:
Ah, bingo...

```
$ ./gradlew -p digdag-standards dependencies | less
  :
+--- com.google.cloud:google-cloud-core:1.94.0
|    +--- com.google.guava:guava:30.0-android (*)
|    +--- com.google.api:gax:1.60.0
|    |    +--- com.google.guava:guava:29.0-android -> 30.0-android (*)
```

Good to know that. Thanks!